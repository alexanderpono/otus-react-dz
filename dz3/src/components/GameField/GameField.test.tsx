import React from 'react';
import { GameField } from './';
import renderer from 'react-test-renderer';

describe('HelloWorld', () => {
    it('It renders using renderer and is compared to snaphot', () => {
        const snapshot = renderer
            .create(<GameField data={['1', '2', '3']} width={60} state={''}></GameField>)
            .toJSON();
        expect(snapshot).toMatchSnapshot();
    });
});
