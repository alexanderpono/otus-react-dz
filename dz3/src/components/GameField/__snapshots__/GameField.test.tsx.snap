// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld It renders using renderer and is compared to snaphot 1`] = `
<section
  className="game-field"
  style={
    Object {
      "width": 60,
    }
  }
>
  <article
    className="game-cell"
    onClick={[Function]}
  >
    <span>
      1
    </span>
  </article>
  <article
    className="game-cell"
    onClick={[Function]}
  >
    <span>
      2
    </span>
  </article>
  <article
    className="game-cell"
    onClick={[Function]}
  >
    <span>
      3
    </span>
  </article>
</section>
`;
